HTML-Classes/HtmlList.php

```diff
@@ -0,0 +1,21 @@
+<?php
+require_once "ElementwithContent.php";
+require_once "ElementwithAttribute.php";
+class HtmlList extends ElementWithContent {
+	protected $content = [ ];
+	private $isOrdered;
+	public function setOrdered($ordered = true) {
+		$this->isOrdered = $ordered;
+	}
+	public function render() {
+		$this->tagname = (($this->isOrdered) ? 'o' : 'u') . 'l';
```

Comment:
readability: replace `(($this->isOrdered) ? 'o' : 'u') . 'l'` with `$this->isOrdered ? 'ol' : 'ul'`

Reply:
```suggestion
		$this->tagname = $this->isOrdered ? 'ol' : 'ul';
```

Reply:
having the `l` separate doesn't really make sense here, since it's part of the tag names